sql/7_0_1-to-7_0_2_upgrade.sql

```diff
@@ -577,4 +577,11 @@ ALTER TABLE `x12_partners` ADD COLUMN `x12_claim_status_endpoint` tinytext;
 
 #IfMissingColumn x12_partners x12_attachment_endpoint
 ALTER TABLE `x12_partners` ADD COLUMN `x12_attachment_endpoint` tinytext;
+#EndIf
+
+#IfNotRow categories name Invoices
+SET @max_rght = (SELECT MAX(rght) FROM categories);
+INSERT INTO categories(`id`,`name`, `value`, `parent`, `lft`, `rght`, `aco_spec`) select (select MAX(id) from categories) + 1, 'Invoices', '', 1, @max_rght, @max_rght + 1, 'encounters|coding' from categories where name = 'Categories';
```

Comment:
This is a bit odd since the above 2 lines used to be done with 1 line:
```
INSERT INTO categories(`id`,`name`, `value`, `parent`, `lft`, `rght`, `aco_spec`) select (select MAX(id) from categories) + 1, 'Invoices', '', 1, rght, rght + 1, 'encounters|coding' from categories where name = 'Categories';
```
(note getting the rght from the Categories entry which is where the max rght should be)
(above way is also ok, but then are doing a select from categories where name - 'Categories' is no longer needed)